SPECS/tdnf/tdnf.spec

```
@@ -129,35 +138,11 @@ mv %{buildroot}/%{_tdnfpluginsdir}/libtdnfrepogpgcheck.so %{buildroot}/%{_tdnfpl
 pushd python
 python3 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
 popd
-find %{buildroot} -name '*.pyc' -delete
-
-# Pre-install
-%pre
-
-    # First argument is 1 => New Installation
-    # First argument is 2 => Upgrade
-
-# Post-install
-%post
-
-    # First argument is 1 => New Installation
-    # First argument is 2 => Upgrade
 
-    /sbin/ldconfig
-
-# Pre-uninstall
-%preun
-
-    # First argument is 0 => Uninstall
-    # First argument is 1 => Upgrade
-
-# Post-uninstall
-%postun
-
-    /sbin/ldconfig
+find %{buildroot} -name '*.pyc' -delete
 
-    # First argument is 0 => Uninstall
-    # First argument is 1 => Upgrade
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
```

We can use these now in 2.0:

```suggestion
%ldconfig_scriptlets
```

---

Great! This makes stuff easier. Thanks!